components/Card/alphaNumberPropTypesValidator.js

```diff
@@ -0,0 +1,19 @@
+export const isValidAlphaNumber = value => {
```

Comment:
![image](https://user-images.githubusercontent.com/3389749/233372260-4e31f212-6193-480a-9205-9fe2b0a5c597.png)

![image](https://user-images.githubusercontent.com/3389749/233372362-e98bdd84-3ed2-492f-8bfd-9829681bddb4.png)

 I think the test simulating error is missing.
WDYT? Make it now or generate a technical  debit?

Reply:
It went unnoticed by me, I'll do it myself, I don't see any problems.